README.md

```diff
@@ -66,6 +67,29 @@ Then to install buildah follow the steps in this example:
   buildah --help
 ```
 
+In Ubuntu/Debian, you can use this command:
+
+```
+  apt-get -qq update
+  apt-get install software-properties-common
+  add-apt-repository ppa:duggan/bats
+  add-apt-repository ppa:alexlarsson/flatpak
+  add-apt-repository ppa:gophers/archive
+  apt-get install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
+  apt-get install golang-1.8
```

Comment:
suggestion:  Change all 'apt-get install' to 'apt-get install -y'.  Also 'add-apt-repository' to 'add-apt-repository -y'.

Reply:
With Ubuntu 17.04.01 server, I'm not able to get skopeo-containers to be found with the above.

I changed the above slightly adding projectatomic repo and moving the update line down  to: 
apt-get install software-properties-common
add-apt-repository ppa:duggan/bats
add-apt-repository ppa:alexlarsson/flatpak
add-apt-repository ppa:gophers/archive
apt-add-repository ppa:projectatomic/ppa
apt-get -qq update
apt-get install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
apt-get install golang-1.8


@lsm5 does this look right to you?

I did this on a vm on my Windoze pc at home, I didn't have to do this on my work RHEL pc.  I may have had the repo for projectatomic already define there and that's why it worked there.